dataland-frontend/src/components/forms/parts/fields/SingleSelectFormField.vue

```diff
@@ -7,7 +7,6 @@
       :validation-label="validationLabel ?? label"
       :placeholder="placeholder"
       :options="options"
-      inner-class="short"
```

Comment:
I think you have to pass "innerClass" to the `SingleSlectFormElement`-component here

Reply:
Clarified this in a call.  We don't pass it because it should be the default in the "Element"-component